docs/conf.py

```diff
@@ -4,16 +4,22 @@
 import sys
 import time
 
+from desktop_notifier import (
```

Comment:
@samschott i did not attempt to generate documentation and am generally unfamiliar with the details of Sphinx etc. so i have no idea if this kind of import can be done here (but if it can it seemed like a good idea to avoid copy/paste of version number etc)